src/resnet.jl

```diff
@@ -10,11 +10,11 @@ Create a basic residual block
                within the residual block
 - `downsample`: set to `true` to downsample the input
 """
-basicblock(inplanes, outplanes, downsample = false) = downsample ?
-  Chain(conv_bn((3, 3), inplanes, outplanes[1]; stride = 2, pad = 1, bias = false)...,
-        conv_bn((3, 3), outplanes[1], outplanes[2], identity; stride = 1, pad = 1, bias = false)...) :
-  Chain(conv_bn((3, 3), inplanes, outplanes[1]; stride = 1, pad = 1, bias = false)...,
+function basicblock(inplanes, outplanes, downsample = false)
+  stride = downsample ? 2 : 1
+  Chain(conv_bn((3, 3), inplanes, outplanes[1]; stride = stride, pad = 1, bias = false)...,
         conv_bn((3, 3), outplanes[1], outplanes[2], identity; stride = 1, pad = 1, bias = false)...)
+end
```

Comment:
I'm not sure this change is equivalent since in the `downsample == false` case, there should only be 1 conv.